torch/distributed/__init__.py

```diff
@@ -67,9 +65,25 @@ def is_available() -> bool:
     )
 
     from .rendezvous import (
+        rendezvous,
         _create_store_from_options,
     )
 
     from .remote_device import _remote_device
 
-    set_debug_level_from_env()
+    _set_debug_level_from_env()
+
+    def _export_c_types():
```

Comment:
I'm unfamiliar with this portion of the PR, could you describe it or maybe someone else can help review?

Reply:
The issue here is that to export symbols they must belong to a module that's part of torch.dist and those are not.
This is the recommended way around to fix it.